mailet/standard/src/main/java/org/apache/james/transport/mailets/LogMessage.java

```diff
@@ -100,25 +187,39 @@ public void service(Mail mail) {
         }
     }
 
+    private void logAttribute(List<String> specificAttributes) {
+        if(!specificAttributes.isEmpty()){
+            logBuffer.append('\n');
+            for(String attribute : specificAttributes){
```

Comment:
Don't we need to read these specific attributes from the email rather than just log the configured value?